Ascend.java

```diff
@@ -2,6 +2,15 @@
 // and prints them in increasing order.
 public class Ascend {
 	public static void main(String[] args) {
-		// Replace this comment with your code
-	}
+		int n = Integer.parseInt(args[0]);
+		int a = (int)(Math.random()*n);

```

Comment:
Lack of spacing between mathematical operations. 
int input1 = (int) (Math.random() * n) 
Please go over the styling guidelines